package.json

```
@@ -15,6 +15,6 @@
     "pkg": "latest"
   },
   "dependencies": {
-    "ldk": "git+https://github.com/open-olive/loop-development-kit-node.git"
+    "ldk": "git+https://github.com/open-olive/loop-development-kit-node.git#SIDE-376-add-config-storage"
```

Maybe remove the specific branch reference? I don't think that branch is present in the LDK repo any more.
```suggestion
    "ldk": "git+https://github.com/open-olive/loop-development-kit-node.git"
```

---

You're absolutely right. Thanks for pointing that out. Removed.